worker-test/pom.xml

```
@@ -170,6 +170,25 @@
                     <startParallel>true</startParallel>
                     <containerNamePattern>%a-%t</containerNamePattern>
                     <images>
+                        <image>
+                            <alias>keystore</alias>
+                            <name>${project.artifactId}-keystore:${project.version}</name>
+                            <build>
+                                <from>${dockerHubPublic}/cafapi/opensuse-jre11:3.0.0</from>
+                                <runCmds>
+                                    <runCmd>mkdir /test-keystore</runCmd>
+                                    <runCmd>openssl genrsa -out /test-keystore/ca_key.pem 2048</runCmd>
+                                    <runCmd>openssl req -x509 -new -key /test-keystore/ca_key.pem -out /test-keystore/ca_certificate.pem -days 3650 -subj "/CN=myname/OU=myorganisational.unit/O=myorganisation/L=mycity/S=myprovince/C=GB"</runCmd>
+                                    <runCmd>openssl genrsa -out /test-keystore/server_key.pem 2048</runCmd>
+                                    <runCmd>openssl req -new -key /test-keystore/server_key.pem -out /test-keystore/server.csr -subj "/CN=myname/OU=myorganisational.unit/O=myorganisation/L=mycity/S=myprovince/C=GB"</runCmd>
+                                    <runCmd>chmod 664 /test-keystore/server_key.pem</runCmd>
+                                    <runCmd>openssl x509 -req -in /test-keystore/server.csr -CA /test-keystore/ca_certificate.pem -CAkey /test-keystore/ca_key.pem -CAcreateserial -out /test-keystore/server_certificate.pem -days 3650</runCmd>
+                                </runCmds>
+                                <volumes>
+                                    <volume>/test-keystore</volume>
+                                </volumes>
+                            </build>
```

Need a run section for arm64 systems that run the images under amd64 emulation
```suggestion
                            </build>
                            <run>
                                <platform>linux/amd64</platform>
                            </run>
```